package.json

```diff
@@ -69,13 +69,14 @@
     "react-router-dom": "^4.3.1"
   },
   "dependencies": {
-    "@getflywheel/local-components": "^17.6.3",
+    "@getflywheel/local-components": "^17.7.1",
     "@reduxjs/toolkit": "^1.4.0",
     "classnames": "^2.2.6",
-    "dateformat": "^3.0.3",
-    "fs-extra": "^9.0.1",
+    "dateformat": "^5.0.3",
+    "fs-extra": "^11.1.1",
+    "glob": "7.2.0",
```

Comment:
This is the same `glob` version that was in the `master` branch:

```sh
# yarn && yarn why glob                      
…
✨  Done in 10.84s.
✨  Done in 16.14s.
yarn why v1.22.19
[1/4] 🤔  Why do we have the module "glob"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "glob@7.2.0"
info Reasons this module exists
   - "@getflywheel#local#awilix" depends on it
   - Hoisted from "@getflywheel#local#awilix#glob"
   - Hoisted from "jest#@jest#core#jest-config#glob"
   - Hoisted from "jest#@jest#core#@jest#reporters#glob"
   - Hoisted from "jest#@jest#core#jest-runtime#glob"
   - Hoisted from "jest#@jest#core#rimraf#glob"
   - Hoisted from "jest#@jest#core#@jest#transform#babel-plugin-istanbul#test-exclude#glob"
info Disk size without dependencies: "64KB"
info Disk size with unique dependencies: "204KB"
info Disk size with transitive dependencies: "300KB"
info Number of shared dependencies: 6
✨  Done in 0.56s.
```

Reply:
Hi @bgturner, is it OK to merge with https://github.com/getflywheel/local-addon-image-optimizer/pull/78/commits/d3c227fa696893f0aae597b5259aee4587ac7649?

Reply:
Nice! Doing some QA and the addon still seems to work. Approved!